vic/drivers/classic/Makefile

```diff
@@ -45,167 +45,54 @@ ifndef CC
 CC=gcc
 endif
 
+# Set the log level
+# To turn off warning statements, set LOG_LVL >= 30
+# | Level     | Numeric value    |
+# |---------  |---------------   |
+# | ERROR     | Always Active    |
+# | WARNING   | < 30             |
+# | INFO      | < 20             |
+# | DEBUG     | < 10             |
+LOG_LVL = 5
+
 # set include
 INCLUDES = -I ${DRIVERPATH}/include -I $(SHAREDPATH)/include -I ${VICPATH}/include
 
 # Uncomment for normal optimized code flags (fastest run option)
 #CFLAGS  = -O3 -Wall -Wno-unused
-LIBRARY = -lm
+# LIBRARY = -lm
 
 # Uncomment to include debugging information
-CFLAGS  =  ${INCLUDES} -g -Wall -Wextra
-#LIBRARY = -lm
+CFLAGS  =  ${INCLUDES} -g -Wall -Wextra -std=c99 -DLOG_LVL=$(LOG_LVL)
+LIBRARY = -lm
 
 # Uncomment to include execution profiling information
-#CFLAGS  = ${INCLUDES} -O3 -pg -Wall -Wno-unused
+#CFLAGS  = ${INCLUDES} -O3 -pg -Wall -Wno-unused -DLOG_LVL=$(LOG_LVL)
 #LIBRARY = -lm
 
 # Uncomment to debug memory problems using electric fence (man efence)
-#CFLAGS  = ${INCLUDES} -g -Wall -Wno-unused
+#CFLAGS  = ${INCLUDES} -g -Wall -Wno-unused -DLOG_LVL=$(LOG_LVL)
 #LIBRARY = -lm -lefence -L/usr/local/lib
 
-# Set the log level
-# To turn off warning statements, set LOG_LVL >= 30
-# | Level     | Numeric value    |
-# |---------  |---------------   |
-# | ERROR     | Always Active    |
-# | WARNING   | < 30             |
-# | INFO      | < 20             |
-# | DEBUG     | < 10             |
-LOG_LVL = 5
+COMPLIB = vic_image
```

Comment:
I'm not sure I understand the choice of the variable name `COMPLIB`. Is this a library?